packages/adapters/src/adapters/Cornerstone3D/MeasurementReport.ts

```diff
@@ -255,7 +255,7 @@ export default class MeasurementReport {
                 "sopCommonModule",
                 imageId
             );
-            const instance = metadataProvider.get("instance", imageId);
+            const instance = metadataProvider.get("instanceModule", imageId);
```

Comment:
`instance` was actually correct i think, since in OHIF we add `instance` and not `instanceModule`. it is kind of an special provider. 
Maybe we should have a fallback or something? 

metadataProvider.get('instance', imageId) | metadataProvider.get('instanceModule', imageId)